pkg/process/cache.go

```diff
@@ -110,14 +119,28 @@ func (pc *Cache) deletePending(process *ProcessInternal) {
 	pc.deleteChan <- process
 }
 
-func (pc *Cache) refDec(p *ProcessInternal) {
+func (pc *Cache) refDec(p *ProcessInternal, reason string) {
+	p.refcntOpsLock.Lock()
+	if val, ok := p.refcntOps[reason]; ok {
+		p.refcntOps[reason] = val + 1
```

Comment:
This would be something like:
```json
"refcntOps": {
  "parent++": 2,
  "parent--": 1,
  "process++": 1
 }
```
which means that we count separately all increase and decrease operations.  So the reason here can be: `process++`, `process--`, `parent++`, or `parent--`.